sockets/challonge.js

```diff
@@ -101,17 +100,28 @@ module.exports = function(io) {
         function getTop8() {
             var top8Object = {
                 matches: [],
-                participants: [],
-                top8Round: top8Round
+                participants: []
             };
             var participantIdList = [];
             var i;
             var match;
             var player;
+            var maxRound = 0;
+            var minRound = 0;
+
+            // This is inefficient, but I couldn't figure out the math formula for determining
+            // which round would put you in the top 8 for winners and losers.
+            for (i = 0; i < matches.length; i++) {
```

Comment:
It looks like this is O(n) unless I'm mistaken, and given the number of matches should be logarithmic relative to number of entrants, this is acceptable for now.